samples/transactional-session/aspnetcore-webapi/SqlPersistenceTS_6/WebApplication/Program.cs

```diff
@@ -11,8 +11,7 @@
 
 public class Program
 {    
-    // for SqlExpress use Data Source=.\SqlExpress;Initial Catalog=nservicebus;Integrated Security=True;Encrypt=false
-    const string ConnectionString = @"Server=localhost,1433;Initial Catalog=nservicebus;User Id=SA;Password=yourStrong(!)Password;Encrypt=false";
+    const string ConnectionString = @"Server=(localdb)\MSSQLLocalDB;Initial Catalog=nservicebus2;Integrated Security=true";
```

Comment:
This should be reverted. As part of moving to SQL in docker we have aligned all connection strings using two examples. One using SqlExpress with integrated security and one with using docker with password based authentication. Encrypt false is also necessary.

Reply:
Relevant PRs

https://github.com/Particular/docs.particular.net/pull/5981
https://github.com/Particular/docs.particular.net/pull/5984
https://github.com/Particular/docs.particular.net/pull/5990

Reply:
that doesn't align with the sample description which states that it's using localdb. Also not sure why we wouldn't use localdb by default rather than SqlExpress? (Encryption settings doesn't seem to be necessary when using localdb)

Reply:
Then let's change that description and link it to the prerequisite documentation we have also updated in the above PRs.

Reply:
Apparently the hint to docker got removed for some reason in the prerequisite files in https://github.com/Particular/docs.particular.net/commit/4c1dd273cbe1f5d69b03fbec2cca05a02931e5f4. Talking to Helen to find out why.